tools/check_installation.py

```diff
@@ -45,6 +53,9 @@ def main(install_dir):
 
     # Check test files detected in repo are installed
     for test_file in scipy_test_files.keys():
+        if test_file in exception_list_test_files:
+            continue
+
         if not test_file in installed_test_files.keys():
```

Comment:
For the linter. Not linked to this PR.
```suggestion
        if test_file not in installed_test_files.keys():
```